app/models/project.server.ts

```diff
@@ -731,3 +731,26 @@ export async function searchProjects({
     ),
   };
 }
+
+export async function deleteProject(projectId: string, profileId: string) {
+  const currentProject = await db.projects.findUniqueOrThrow({
+    where: { id: projectId },
+    select: {
+      name: true,
+      ownerId: true,
+    },
+  });
+
+  const projectMembers = await db.projectMembers.findMany({
+    where: { id: profileId },
+    select: {
+      profileId: true,
+    },
+  });
+  console.log(currentProject);
+
+  validateIsTeamMember(profileId, projectMembers, currentProject.ownerId);
```

Comment:
Based on this line, in order to delete a project, you must be an Admin and a Project Member, is that correct? If I'm an Admin I must join the project first before deleting it? I'm asking before I didn't attend the stand ups where this was discussed so I don't know what's the expected flow